packages/decoupled-logger/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "decoupled-logger",
+  "version": "1.0.1",
+  "description": "decoupled logger.",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "build": "scripts/build.sh",
+    "install": "scripts/build.sh",
+    "watch": "tsc -p tsconfig.dev.json --watch",
+    "lint": "tslint -t verbose src/**/*.ts{,x}",
+    "test": "jest dist"
+  },
+  "engines": {
+    "node": "node >= 8.0.0"
+  },
+  "dependencies": {
+    "@types/jest": "^22.2.3",
+    "@types/lodash": "^4.14.109",
+    "@types/winston": "^2.3.9",
+    "body-parser": "^1.18.2",
```

Comment:
@friendken Please check dependencies - do we really need those? 
body-parser, node-cache, node-fetch, regenerator-runtime, sinon...